### PR TITLE
Nucleotide-count hints add

### PR DESCRIPTION
**Pull request by jmbradnan**
Added HINTS.md file to nucleotide-count exercise (#209)

### CODE REVIEW

File: exercises/nucleotide-count/HINTS.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+##HINTS
+For this exercise the following C# feature comes in handy:
```
Comment (ErikSchierboom) [inline exercises/nucleotide-count/HINTS.md]: Actually, this exercise requires the use of a Dictionary, so I would rephrase:

"This exercise requires you to return a dictionary. For more information, see [this page](https://msdn.microsoft.com/en-us/library/s4ys34ea(v=vs.110).aspx)."

File: exercises/nucleotide-count/HINTS.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+##HINTS
```
Comment (ErikSchierboom) [inline exercises/nucleotide-count/HINTS.md]: Please add a space between the hashes and the text, and also use non-caps text (so that would become "Hints")

File: exercises/nucleotide-count/HINTS.md (hunk 1 above)
Comment (jmbradnan) [inline exercises/nucleotide-count/HINTS.md]: I made the changes above and resubmitted (did a new branch and followed the steps you list above so hopefully only the one change is present).

**Comment by ErikSchierboom** [comment]
Closed with #276

File: exercises/nucleotide-count/HINTS.md (hunk 1 above)
Comment (ErikSchierboom) [inline exercises/nucleotide-count/HINTS.md]: @jmbradnan Yep! Everything was perfect so it has been merged. Thanks
